packages/functional-tests/tests/misc/cookiesDisabled.spec.ts

```diff
@@ -24,13 +24,13 @@ test.describe('cookies disabled', () => {
 
     //Verify the Cookies disabled header
     await page.waitForURL(/\/cookies_disabled/);
-    await cookiesDisabled.waitForCookiesDisabledHeader();
+    expect(await cookiesDisabled.cookiesDisabledHeader()).toBeVisible();
```

Comment:
Interesting!
The `cookiesDisabledHeader()` has an `await header.waitFor();`. 
Does the `.waitFor` ensure the element is in the DOM and is visible, or just in the DOM? No harm in also checking it is visible here too, just curious why both needed.

Reply:
It waits for a little bit longer before it can say visible or not. And this particular header was taking a bit longer to wait hence I used the waitFor().